README.md

```diff
@@ -41,7 +43,21 @@ Once you run synthtool without errors:
 - commit and push the changes to a branch and open a PR!
 
 ### Node.js
-- TODO. Should be similar to Python
+Node.js generation is similar to python. For an example `synth.py` you can look at https://github.com/googleapis/nodejs-speech/blob/master/synth.py.
 
 ### Ruby
 - TODO. Should be similar to Python
+
+
+## Features
+### Templating
+Synthtool supports template files using jinja. As an example let's look at node.js. The templates for node can be found at `/synthtool/gcp/templates/node_library/`. As of writing this doc, the following works in a node.js synth script.
```

Comment:
Lets leave out the `As of writing this doc`.  Hope is that we keep the README up to date as we make changes :)